### PR TITLE
feat(Workflow) Add ScanBarCode MetaVariable

### PR DESCRIPTION
**Pull request by mocodertz**

### CODE REVIEW

File: modules/com_vtiger_workflow/VTSimpleTemplate.inc [hunk 1]
```diff
@@ -593,6 +596,36 @@ class VTSimpleTemplate {
...
+					} elseif (array_key_exists($codetype, $tcpdf2dEncodingTypes)) {
+						$barcodeobj = new TCPDF2DBarcode($fieldValue, $codetype);
+					}
+					$img = $barcodeobj->getBarcodePngData();
+					$fname = uniqid('barcode');
+					@unlink('cache/images/'.$fname.'.png');
+					file_put_contents('cache/images/'.$fname.'.png', $img);
+					return '<img src="cid:'.$fname.'" />';
+					print_r('<img src="cid:'.$fname.'" />');
+					die();
```
Comment (joebordes) [inline modules/com_vtiger_workflow/VTSimpleTemplate.inc]: please eliminate these two lines and I will accept

**Comment by joebordes** [comment]
wow looks like a lot of work, much more than I had imagined. great job!  thanks

**Comment by joebordes** [comment]
ah, next time separate code formatting changes from actual functionality changes, put the code formatting in its own commit to make it easier to debug.

File: modules/com_vtiger_workflow/VTSimpleTemplate.inc (hunk 1 above)
Comment (mocodertz) [reply modules/com_vtiger_workflow/VTSimpleTemplate.inc]: Done

**Comment by joebordes** [comment]
thanks!

**Pull request merged**